src/main/java/uk/gov/hmcts/darts/common/entity/NodeRegisterEntity.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.hmcts.darts.common.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.SequenceGenerator;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Table(name = NodeRegisterEntity.TABLE_NAME)
+@Getter
+@Setter
+public class NodeRegisterEntity {
```

Comment:
I think this will need to `implements JpaAuditing` to add the created / modified audit columns

Reply:
Currently this table doesn't have the extra audit columns